blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/AbstractPipelineImpl.java

```diff
@@ -178,23 +181,15 @@ public Link getLink() {
         return OrganizationImpl.INSTANCE.getLink().rel("pipelines").rel(getRecursivePathFromFullName(this));
     }
 
-    public static String getRecursivePathFromFullName(BluePipeline pipeline){
```

Comment:
@tfennelly This method should not change, it computes recursive path from full name. This expects pipeline.getFullName() has already been encoded properly and no changes needed. So if there is pipeline named 'p/p1' inside folder named 'f/f1'. folderPipeline.getFullName() should give `f%2Ff1/p%2Fp1`, this would result in return from this method as `/pipelines/f%2Ff1/pipelines/p%2Fp1`.